docs/src/content/docs/blog/introducing-lunaria.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Introducing Lunaria
+excerpt: 
+  Lunaria is a complete localization management toolchain built especially for open-source projects. Currently, Lunaria is already being used in production to help track over 5000 content files in over a dozen of locales in big projects such as [Astro](https://i18n.docs.astro.build/), [Starlight](https://i18n.starlight.astro.build/), [SolidJS](https://docs.solidjs.com/i18n-status/), and [VitePress](https://vitepress.dev/_translations/).
+date: 2024-05-21
+authors:
+  name: Yan Thomas
+  title: Maintainer
+  picture: https://avatars.githubusercontent.com/u/61414485
+  url: https://github.com/yanthomasdev/
+---
+
+Maintaining a project for an international audience is no easy task!
+
+It is a lot of effort to build a sustainable workflow for content to be progressively and consistently localized, especially if you also have contributors to your project.
+
+Lunaria is a complete **localization management toolchain** built especially for open-source projects. 
+
+Currently, Lunaria is already being used in production to help track over 5000 content files in over a dozen of locales in big projects such as [Astro](https://i18n.docs.astro.build/), [Starlight](https://i18n.starlight.astro.build/), [SolidJS](https://docs.solidjs.com/i18n-status/), and [VitePress](https://vitepress.dev/_translations/).
+
+:::tip[Ready to get started?]
+Read the official ["Getting Started" guide](https://lunaria.dev/getting-started/) to learn more about and set up Lunaria in your project. 
+:::
+
+## Why Lunaria?
+
+During the process of making the Astro documentation available for translation, we noticed how hard would be to make a sustainable workflow for both maintainers and contributors. First, most localization management platforms (the ones that help you keep track of and update localizations) have the requirement of owning your content, are paid to some degree, and more importantly, are way too complex for beginners and might not even be available for people in certain countries one might want to reach.
+
+With all this in mind, the Astro maintainer [`@hippotastic`](https://github.com/hippotastic) built what was then called "Translation Tracker", a tailor-made translation tracking tool that allowed us to work alongside contributors to translate the official Astro documentation in 14 different languages with ease.
+
+Lunaria is the evolution of said tool, with a wider scope, more features, and support for different types of projects -- fun fact, Lunaria was first hinted at in the ["How Astro does i18n" blog post](https://astro.build/blog/astro-i18n/#going-forward)!
+
+## What's in a "toolchain"?
+
+Lunaria is a *toolchain*, a set of tools that **supercharge your localization workflow** by integrating several parts of your project's infrastructure, including your repository's Git history, your project's file structure, and development platform such as GitHub and GitLab.
```

Comment:
I _think_ “integrating” needs a preposition? Either “with” or “into”?

Also nit: I guess “or” is more appropriate at the end?

```suggestion
Lunaria is a *toolchain*, a set of tools that **supercharge your localization workflow** by integrating with several parts of your project's infrastructure, including your repository's Git history, your project's file structure, and development platform such as GitHub or GitLab.
```